.pipelines/templates/template-prod-e2e-steps.yml

```diff
@@ -24,6 +24,7 @@ steps:
     . ./hack/e2e/run-rp-and-e2e.sh
 
     export CI=true
+    export ARO_IMAGE="arosvc.azurecr.io/aro:$(get_e2e_image_version)"
```

Comment:
I think this file can run in INT and PROD context therefore hardcoding to arosvc must be wrong?  Maybe you need to make RP_IMAGE_ACR available in this context?